cli/operations.py

```diff
@@ -17,62 +17,78 @@
 
 azure_storage_client = None
 
+
 class ImageLimitException(Exception):
     pass
 
 
 def supported_file_type(file_name):
+    if file_name.startswith('.'):
+        return False
+
     file_suffix = pathlib.Path(file_name).suffix.lower()
     if file_suffix in ['.png', '.jpg', '.jpeg', '.gif']:
         return True
     else:
         return False
 
 
+# Somewhat of a hack to remove the folder name passed on the
```

Comment:
@bnookala what does an example of this output look like?

Reply:
@andrebriggs I added this to the code file as documentation, but here it is again for posterity:

# Example invocation: python3 -m cli onboard /my/full/path
# os.walk will return: /my/full/path/1.jpg, /my/full/path/2.jpg, etc.